PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -352,6 +353,13 @@ private extension CollectionViewModel {
         return snapshot
     }
 
+    static func emptySnapshot() -> Snapshot {
```

Comment:
do we want to add a clarifying comment here?

Reply:
for sure!